WcaOnRails/app/models/vote.rb

```diff
@@ -37,7 +37,7 @@ def number_of_options_must_match_poll
 
   validate :poll_must_be_confirmed
   def poll_must_be_confirmed
-    if poll && !poll.confirmed
+    if poll && poll.confirmed_at.nil?
```

Comment:
I'd like to see the change, if you don't mind. It will keep the diff down, and save future readers of the code some thinking.